### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by matejthetree**
Added async / await example to requester and receiver

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -566,6 +566,27 @@ makeRequest(req)
...
+    val: Math.floor(Math.random() * 10),
+};
+
+async function  makeRequest () {
+  const response = await randomRequester.send(req);
+  console.log(response);
+}
+
+makeRequest(req)
+    .then(process.exit);
```
Comment (dashersw) [inline README.md]: I think it's better not to mix promises with async/await. Could you put the process exit in the `makeRequest` function? Or another `async function main` that awaits `makeRequest` and then exits.

File: README.md [hunk 2]
```diff
@@ -630,6 +651,34 @@ userRequester
...
+userResponder.on('sum', async (req) => req.a + req.b);
+```
+
+`requester.js`
+```js
+const cote = require('cote');
+const sumRequester = new cote.Requester({ name: 'Sum Requester' });
+
+async function requestSum() {
+  const sum = await userRequester.send({ type:'sum', a:3, b:5 });
```
Comment (dashersw) [inline README.md]: It would be good to use the exact example in promises above, instead of a sum operation.

File: README.md [hunk 3]
```diff
@@ -630,6 +651,34 @@ userRequester
...
+  console.log(sum);
+}
+
+requestSum()
+    .then(process.exit);
```
Comment (dashersw) [inline README.md]: Here again, the process exit shouldn't come in a `then`.

File: README.md (hunk 1 above)
Comment (matejthetree) [reply README.md]: Sure

**Pull request merged**